### PR TITLE
update to wechaty-puppet-wechat 1.18.4

### PR DESCRIPTION
**Pull request by choogoo**
## I'm submitting a...

```
[ ] Bug Fix
[ ] Feature
[- ] Other (Refactoring, Added tests, Documentation, ...)
```

## Checklist

- [ ] Commit Messages follow the [Conventional Commits](https://conventionalcommits.org/) pattern
  - A feature commit message is prefixed "feat:"
  - A bugfix commit message is prefixed "fix:"
- [ ] Tests for the changes have been added

## Description

_please describe the changes that you are making_

_for features, please describe how to use the new feature_

_please include a reference to an existing issue, if applicable_

## Does this PR introduce a breaking change?

```
[ ] Yes
[- ] No
```

### CODE REVIEW

File: examples/ding-dong-bot.ts
Comment (huan) [inline examples/ding-dong-bot.ts:79]: Could you please use English to write the comments because it will be more readable by developers all over the world?

Thank you very much!

File: examples/ding-dong-bot.ts
Comment (huan) [inline examples/ding-dong-bot.ts:81]: Please always use `ding-dong-bot` as the name of our getting started bot.

**Review by huan — CHANGES_REQUESTED** [review]
Thank you very much for helping the community upgrade the latest version!

This PR will be good to merge after following my reviews.

File: examples/ding-dong-bot.ts
Comment (choogoo) [reply examples/ding-dong-bot.ts]: OK, it's my negligence. It has been revised

File: examples/ding-dong-bot.ts
Comment (choogoo) [reply examples/ding-dong-bot.ts]: done

**Comment by huan** [comment]
Thank you very much for your contribution!

## You are welcome to join [Wechaty Contributor Program](https://wechaty.js.org/docs/contributor-program/)

### 1. Join Wechaty Organization

> You've invited choogoo to Wechaty! They'll be receiving an email shortly. They can also visit https://github.com/wechaty to accept the invitation.

I have invited you to join our Wechaty GitHub Organization, please accept it by following the above message. (See also: [wechaty/PMC#16](https://github.com/wechaty/PMC/issues/16))

### 2. Update Your Wechaty Contributor Profile

1. Please open [Contributor Hall of Fame](https://docs.google.com/spreadsheets/d/1XcDoIczyIclqXP1p90Sz7S0n4Q22xjFzJmjPFlU2g1E/edit#gid=0) and add yourself to the end of the list, so that other contributors will know you better!
1. Please add yourself to our Website [Contributors](https://wechaty.js.org/contributors/) by [creating a PR](https://github.com/wechaty/jekyll/tree/main/jekyll/_contributors) and refer to this PR link as well.

### 3. Join The Contributor Only WeChat Room

We also have a WeChat room for contributors only which can discuss Wechaty at a deeper level, you are welcome to join and if you are interested.

Please add @lijiarui wechat: _ruirui_0914_ and send her this pr link. She will invite you into Wechaty Contributor Room

Cheers!

**Review by huan — CHANGES_REQUESTED** [review]
Please make sure the following requirements have been met for your PR before we can merge it:

1. The PR has passed all the CI tests and turned _green_. (you can execute all tests locally by running `npm test`)

**Review by choogoo — COMMENTED** [review]
Fix wechaty-popup-service quick run commands and add dependency packages.

**Pull request closed without merging**
